src/components/primitive/box/box.component.tsx

```diff
@@ -1,156 +1,154 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Col, Row } from 'react-grid-system';
-import styled, { withTheme } from 'styled-components';
+import styled, { ThemeContext } from 'styled-components';
 import _ from 'lodash';
 import { BoxComponentProps } from './box.component.props';
 import { spacing } from '../../../utils';
-import { IStyledTheme } from '../../../themes/styled-components';
 
-// @ts-ignore
-export const Box: React.FunctionComponent<BoxComponentProps> = withTheme(
-  (props: BoxComponentProps & IStyledTheme) => {
-    const {
-      theme: { colors },
-      row,
-      borderRadius,
-      spaceBetween,
-      flex,
-      hidden,
-      bottom,
-      children,
-      color,
-      height,
-      width,
-      maxHeight,
-      maxWidth,
-      hCenter,
-      vCenter,
-      marginTop,
-      marginBottom,
-      marginLeft,
-      marginRight,
-      marginHorizontal,
-      marginVertical,
-      margin,
-      paddingTop,
-      paddingBottom,
-      paddingLeft,
-      paddingRight,
-      paddingHorizontal,
-      paddingVertical,
-      padding,
-      wrap,
-      rightAlign,
-      centerAlign,
-      style: styleOverride,
-      ...rest
-    } = props;
+export const Box: React.FunctionComponent<BoxComponentProps> = (
+  props: BoxComponentProps
+) => {
+  const { colors } = useContext(ThemeContext);
```

Comment:
Where is this coming from? In order to use the theme, you need to wrap the component with withTheme HOC.